mmdet/models/roi_heads/roi_extractors/single_level.py

```diff
@@ -84,7 +92,9 @@ def roi_rescale(self, rois, scale_factor):
         y2 = cy + new_h * 0.5
         new_rois = torch.stack((rois[:, 0], x1, y1, x2, y2), dim=-1)
         return new_rois
-
+    
+    # adapter=lambda self, feats, rois: ((rois,) + tuple(feats), {"output_size": self.roi_layers[0].out_size[0], "featmap_strides": self.featmap_strides, "sample_num": self.roi_layers[0].sample_num})
+    @py_symbolic(op_name='roi_feature_extractor', adapter=adapter)
```

Comment:
Lambda becomes too heavy as it handles keyword-argument. So, please, remove the commented code.